### DISCUSSION TITLE
getting Not authorized. for anonymous login

### DISCUSSION THREAD

**Discussion by Himanshu-Dodwani** [umputun/remark42#1762] [Q&A]
<img width="1070" alt="image" src="https://github.com/umputun/remark42/assets/135614444/129c8578-0255-458d-a227-7b3e90a1bb99">
this is what i am getting when i am adding a comment from anonymous.
<img width="483" alt="image" src="https://github.com/umputun/remark42/assets/135614444/1a6b69f0-b5fd-40bd-9003-6df3f0c8adec">
<img width="539" alt="image" src="https://github.com/umputun/remark42/assets/135614444/9b769852-4251-4b9e-9e2f-a6000f75b48d">

**Comment by angristan** [comment]
I'm also encountering this, did you find a solution?
